### PR TITLE
Switching to Paella player by default

### PR DESCRIPTION
**Pull request by gregorydlogan**
This PR sets the default player to Paella, as discussed in the developer meeting.

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] be against the correct branch (features can only go into develop)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated testing
* [x] have a clean commit history
* [x] have proper commit messages (title and body) for all commits
* [x] have appropriate tags applied

### CODE REVIEW

File: docs/guides/admin/docs/modules/mediamodule.configuration.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ The configurations for the media module are done for each tenant. So the configu
     - Default: Opencast logo
 - `prop.player`
     - The player that should be use to play the videos.
-    - Default: Theodul player
+    - Default: Paella player
```
Comment (lkiesow) [inline docs/guides/admin/docs/modules/mediamodule.configuration.md:14]: Interesting. This configuration does not exist any longer.
The only part which still exists is:

https://github.com/opencast/opencast/blob/d2ab45a9fd345b4bd3fe7bc5160de6446b2d91f2/etc/ui-config/mh_default_org/engage-ui/config.yml#L1-L3

For the player, the media module uses the `/play` endpoint which automatically redirects to the default player.

File: docs/guides/admin/docs/modules/paella.player/configuration.md [hunk 2]
```diff
@@ -12,17 +12,7 @@ Paella has been specially designed for lecture recordings. It works with all HTM
...
-
-To enable paella player you need to edit the `prop.player` variable.
-This can be enabled for each tenant. So the configuration keys are located in
-`etc/org.opencastproject.organization-mh_default_org.cfg`.
-
-
-To activate the paella player set:
-
-    prop.player=/paella/ui/watch.html?id=#{id}
+Starting with Opencast 9.0 Paella is Opencast's default player
```
Comment (lkiesow) [inline docs/guides/admin/docs/modules/paella.player/configuration.md]: This is the 9+ documentation, so:

```suggestion
Paella is Opencast's default player.
```

Version numbers get old ;-)

**Review by lkiesow — CHANGES_REQUESTED** [review]
Generally looks good. Minor documentation change request.

File: docs/guides/admin/docs/modules/mediamodule.configuration.md (hunk 1 above)
Comment (gregorydlogan) [reply docs/guides/admin/docs/modules/mediamodule.configuration.md:14]: Which loads this config item, doesn't it?  That's what the code looks like at any rate.

File: docs/guides/admin/docs/modules/paella.player/configuration.md (hunk 2 above)
Comment (gregorydlogan) [reply docs/guides/admin/docs/modules/paella.player/configuration.md]: Hah, fair :)

File: docs/guides/admin/docs/modules/mediamodule.configuration.md (hunk 1 above)
Comment (lkiesow) [reply docs/guides/admin/docs/modules/mediamodule.configuration.md:14]: I'm slightly confused. The [docs you adjusted](https://github.com/opencast/opencast/blob/cb19e3c304404795a04a225feeadfade4aa0cf24/docs/guides/admin/docs/modules/mediamodule.configuration.md) is supposed to describe the media module configuration which does not contain a player configuration and now reside at a different location.

Comment (gregorydlogan) [reply docs/guides/admin/docs/modules/mediamodule.configuration.md:14]: I found [this](https://github.com/opencast/opencast/blob/cb19e3c304404795a04a225feeadfade4aa0cf24/modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java) which makes it look to me like it's still in use.  Whether that's true or not I guess I didn't actually check, but it looks like it should be in use...

Comment (lkiesow) [reply docs/guides/admin/docs/modules/mediamodule.configuration.md:14]: It is. But not in the media mudule :D 
Anyway, let's get it in like this. The other parts are something we can fix later.

**Pull request merged**